config/locales/en.yml

```diff
@@ -20,65 +20,63 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
```

Comment:
I appreciate the clean up, but similar to before, in the future can we keep only the relevant changes in the commit.

Reply:
Will do. Honestly don't remember doing that...